Improve workflow document (#163)
TheBevyFlock/bevy_quickstart: docs/workflows.md

### Set up variables

> [`.github/workflows/release.yaml`](../.github/workflows/release.yaml) contains a list of environment variables that are setup automatically for you if you use `cargo generate`.

Reviewer: ```suggestion
[`.github/workflows/release.yaml`](../.github/workflows/release.yaml) contains a list of environment variables that are set automatically by `cargo generate`.
```

Otherwise, you will have to fill them in manually and push a commit.

<details>

> <summary>Relevant section of the release workflow</summary>

Reviewer: ```suggestion
<summary>Environment variables section of the release workflow</summary>
```

### Set up itch.io upload

Create your itch.io page with the same name as is used in the `ITCH_TARGET` variable in [release.yaml](../.github/workflows/release.yaml).
By default, this is the same as the project name.
Also set `Kind of project` to `HTML`.

> After creation, you'll see a warning saying "There was a problem loading your project: No file provided to embed". Don't worry about it, we will fix that in a moment.

Reviewer: ```suggestion
On your project page, you'll see a warning saying "There was a problem loading your project: No file provided to embed". This is because we haven't uploaded a release yet.
```

Author: Not a fan of the second part, because it implies we will fix that right now. In fact, we first do the credential dance.

Author: Counteroffer: [`2ccdaa0` (#163)](https://github.com/TheBevyFlock/bevy_quickstart/pull/163/commits/2ccdaa045b6d2bf6f40ecf708dc8e8531431910d)

### Set up itch.io upload

> Create your itch.io page with the same name as is used in the `ITCH_TARGET` variable in [release.yaml](../.github/workflows/release.yaml).

Reviewer: Should still specify `Kind of project = HTML`

Author: Whoops

Author: [`6021871` (#163)](https://github.com/TheBevyFlock/bevy_quickstart/pull/163/commits/60218717460e7337d668dd0a1d78b6b067fc4159)
